x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/create_step/create_step.tsx

```diff
@@ -75,125 +64,63 @@ export const CreateStep: FC<Props> = ({ actions, state, step }) => {
-                    <EuiCheckbox
-                      disabled={isJobCreated || canCreateDataView === false}
-                      name="mlDataFrameAnalyticsCreateDataView"
-                      id={'dataframe-create-data-view-checkbox'}
-                      label={i18n.translate(
-                        'xpack.ml.dataframe.analytics.create.createDataViewLabel',
-                        {
-                          defaultMessage: 'Create data view',
-                        }
-                      )}
-                      checked={createDataView === true}
-                      onChange={() => setFormState({ createDataView: !createDataView })}
-                      data-test-subj="mlAnalyticsCreateJobWizardCreateDataViewCheckbox"
-                    />
-                  </EuiFormRow>
-                </EuiFlexItem>
-              ) : null}
-            </EuiFlexGroup>
-          </EuiFlexItem>
-          <EuiFlexItem grow={false}>
-            <EuiButton
-              className="mlAnalyticsCreateWizard__footerButton"
-              disabled={
-                !isValid ||
-                !isAdvancedEditorValidJson ||
-                (destinationDataViewTitleExists === true && createDataView === true)
-              }
-              onClick={handleCreation}
-              fill
-              isLoading={creationTriggered}
-              data-test-subj="mlAnalyticsCreateJobWizardCreateButton"
-            >
-              {i18n.translate('xpack.ml.dataframe.analytics.create.wizardCreateButton', {
-                defaultMessage: 'Create',
-              })}
-            </EuiButton>
-          </EuiFlexItem>
-        </EuiFlexGroup>
+        <>
+          {showCreateDataView && (
+            <>
+              <CreateDataViewForm
+                canCreateDataView={canCreateDataView}
+                createDataView={createDataView}
+                dataViewTitleExists={destinationDataViewTitleExists}
+                setCreateDataView={() => setFormState({ createDataView: !createDataView })}
+                dataViewAvailableTimeFields={dataViewAvailableTimeFields}
+                dataViewTimeField={timeFieldName}
+                onTimeFieldChanged={onTimeFieldChanged}
+              />
+              <EuiSpacer />
+            </>
+          )}
+
+          <EuiFlexGroup gutterSize="m" alignItems="center">
+            <EuiFlexItem grow={false}>
+              <EuiFormRow
+                helpText={i18n.translate(
+                  'xpack.ml.dataframe.analytics.create.startCheckboxHelpText',
+                  {
+                    defaultMessage:
+                      'If unselected, job can be started later by returning to the jobs list.',
+                  }
+                )}
+              >
+                <EuiSwitch
+                  data-test-subj="mlAnalyticsCreateJobWizardStartJobSwitch"
+                  name="mlAnalyticsCreateJobWizardStartJobSwitch"
+                  disabled={canCreateDataView === false}
```

Comment:
Why is the enabled state of this switch linked to `canCreateDataView`?

Reply:
Good find, thanks! That was a copy/paste mistake, the original checkbox didn't have a `disabled` attribute so I removed it again from the switch in [0b8692e](https://github.com/elastic/kibana/pull/171202/commits/0b8692ea132db91593f7efe7ef55272273b469c0).